test/judges/test-github-events.rb

```diff
@@ -641,6 +636,14 @@ def test_pull_request_event_with_comments
     assert_equal(1, f.comments_resolved)
   end
 
+  def test_count_numbers_of_workflow_builds
+    fb = Factbase.new
+    load_it('github-events', fb, Judges::Options.new({ 'repositories' => 'foo/foo', 'testing' => true }))
```

Comment:
@Suban05 Why not use web mock for this cases? Testing mode used for run yml files, for example - https://github.com/zerocracy/judges-action/blob/master/judges/github-events/quick-scan.yml

Reply:
@Yegorov good question. For the `pull-was-merged` event,  we use `github_graph`. Maybe I don't understand something, but it's very difficult to mock this thing by `WebMock`. And I use a fake client for `Fbe.github_graph`. However, there is an `options` argument  in both clients (`Fbe.octo` and `Fbe.github_graph`), and when I set `testing` globally, `Fbe.octo` also becomes a fake client. That's why I use fake clients